src/Jab/ContainerGenerator.cs

```diff
@@ -276,6 +280,7 @@ private void Execute(GeneratorContext context)
                             codeWriter.Line($";");
                         }
 
+                        codeWriter.LineRaw(_generatedCodeAttribute);
```

Comment:
Do you need the attribute for nested classes?

Reply:
I think it's not necessary

Reply:
Removed